src/System.Composition.AttributedModel/src/System/Composition/Convention/AttributedModelProvider.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
```

Comment:
I view unused using as dead code, and dead code should be removed.  Doesn't seem like a style thing to me, personally.